src/styles/button.scss

```diff
@@ -60,6 +60,7 @@ $block: #{$fd-namespace}-button;
   line-height: 1rem;
```

Comment:
Looks broken 

<img width="367" alt="Screenshot 2021-11-04 at 12 41 44" src="https://user-images.githubusercontent.com/20265336/140300353-6e7c5f12-59af-4015-b9c5-cd27bb995fb7.png">

Reply:
Will be implementing the segmented button in my next PR